css/_icon-fonts.scss

```diff
@@ -81,4 +82,7 @@
 }
 .icon-youtube:before {
   content: "\e915";
+}
+.icon-menu:before {
```

Comment:
Can we move this icon to the right side? [https://prnt.sc/v52jay](https://prnt.sc/v52jay)